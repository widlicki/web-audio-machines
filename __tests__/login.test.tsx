import { render } from '@testing-library/react'
import Login from '../pages/sign-in'

describe('Login', () => {
  it('renders login page according to snapshot', () => {
    const { container } = render(<Login />)
    expect(container).toMatchSnapshot()
  })
})
