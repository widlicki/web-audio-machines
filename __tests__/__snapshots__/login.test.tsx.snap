// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login renders login page according to snapshot 1`] = `
<div>
  <div>
    <h1>
      Login Page
    </h1>
  </div>
</div>
`;
